app/src/main/java/org/p2p/wallet/jupiter/interactor/JupiterSwapInteractor.kt

```diff
@@ -47,9 +46,10 @@ class JupiterSwapInteractor(
         Timber.i("Swap tokens success: $firstTransactionSignature")
         JupiterSwapTokensResult.Success
     } catch (error: ServerException) {
-        val errorMessage = error.message.orEmpty()
-        if (LOW_SLIPPAGE_ERROR_CODE in errorMessage ||
-            LOW_SLIPPAGE_ERROR_MESSAGE in errorMessage
+        if (error.domainErrorType != null &&
```

Comment:
you can move error.domainErrorType to variable

Reply:
done